### DISCUSSION TITLE
When is the next release (1.14)?

### DISCUSSION THREAD

**Discussion by James-ZHANG** [Yubico/libfido2#737] [General]
Hi,
    If I understand correctly the release cycle for libfido2 is about one minor version per half a year. However, there hasn't been a release tag since 02/2023.
    When do we plan to do the 1.14 release?

**Comment by LDVG** [comment]
Hi,

Our current goal is to release 1.14.0 sometime before the end-of-year. Unfortunately, we cannot be more precise than that. We will update this thread when the release is out.

Thank you for your interest in libfido2!

**Comment by LDVG** [comment]
Hi again @James-ZHANG,

1.14.0 was released 2023-11-13.

Have a nice day!
